documentation/service_admin/installing-openshift.adoc

```diff
@@ -83,7 +83,14 @@ oc create secret generic keycloak-credentials --from-literal=admin.username=admi
 +
 [options="nowrap"]
 ----
-oc adm policy add-role-to-user view system:serviceaccount:$(oc project -q):default
+oc adm policy add-role-to-user view system:serviceaccount:enmasse:default
+----
+
+. Create keycloak configuration used by controller and service
++
+[options="nowrap"]
+----
+oc create configmap keycloak-config --from-literal=hostname=standard-authservice --from-literal=amqpPort=5671 --from-literal=httpPort=8443 --from-literal=caSecretName=standard-authservice-cert --from-literal=oauthURL=https://keycloak-enmasse.127.0.0.1.nip.io:443/auth
```

Comment:
I don't think this will work, as 'keycloak-enmasse.127.0.0.1.nip.io' will resolve to 127.0.0.1 in the admin pod.